charts_v3/helm-broker/README.md

```diff
@@ -0,0 +1,7 @@
+# Helm Broker
+
+## Overview
+
+The Helm Broker is a [Service Broker](https://kyma-project.io/docs/master/components/service-catalog/#service-brokers-overview) which exposes Helm charts as Service Classes in the [Service Catalog](https://kyma-project.io/docs/master/components/service-catalog/#overview-overview). To do so, the Helm Broker uses the concept of addons. An addon is an abstraction layer over a Helm chart which provides all information required to convert the chart into a Service Class.
+
+The Helm Broker implements the [Open Service Broker API](https://github.com/openservicebrokerapi/servicebroker/blob/master/spec.md). For more information about the Helm Broker, read the [documentation](https://kyma-project.io/docs/master/components/helm-broker/).
```

Comment:
I don't think we should link to the documentation on the website from master. I switched it to link to the default docs (latest release).
If you mean to link to the latest documentation on master, we should probably link to the repository directly. If that was not an oversight, please let's discuss it.
```suggestion
The Helm Broker implements the [Open Service Broker API](https://github.com/openservicebrokerapi/servicebroker/blob/master/spec.md). For more information about the Helm Broker, read the [documentation](https://kyma-project.io/docs/components/helm-broker/).
```

Reply:
As discussed, let's leave it as it is for now after all.